### ISSUE TITLE
[SF][A11Y][ui5-select]: Option value is not announced by screen reader

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#5682]
Hi Colleague,
Here is a a11y ticket raised by QAS team: option value is not announced by screen reader in Select component. Please help take a look.

https://user-images.githubusercontent.com/78520316/185138014-c2798863-7a59-42f8-83a3-b1ffa45186bf.mp4

### **Steps to Reproduce**
 1. Login to QACAND: https://qacand.hcm.ondemand.com/sf/omp/?mock#/manage
2. Select "Bookmarks" or "In Progress" or "History" tab
3. Focus to select besides "Filter by"
4. Open drop select options
5. Listen to Screen Reader

### **Context**
- UI5 Web Components version: {1.6.0}
- Affected component: {ui5-select}

### **Priority**
- [ ] Low
- [] Medium
- [x ] High
- [ ] Very High

Thanks a lot,
Josephine Su

**Comment by unazko** [comment]
Hello Hello @SAP/ui5-webcomponents-topic-p,

The ui5-select list items aren't announced by the screen reader when navigating between.
The issue is reproducible via JAWS 2021, 2022 and NVDA.

Not sure if the focus resides on the list items during navigation or is just a visual outline.
As this could be causing the issue if the actual focus isn't over a specific item.

Could you please take a look.

Best Regards,
Boyan

**Comment by dobrinyonkov** [comment]
duplicate: #5494
